index.html

```diff
@@ -153,6 +153,21 @@ <h3><dfn>CropTarget</dfn> Definition</h3>
           };
         </pre>
         <dl data-link-for="CropTarget" data-dfn-for="CropTarget"></dl>
+        <p>
+          To <dfn data-export>create a CropTarget</dfn> with <var>element</var> as input, run the following steps:
+          <ol>
+            <li>
+              <p>Let <var>cropTarget</var> be a new object of type {{CropTarget}}.</p>
+            </li>
+            <li>
+              <p>Create <var>cropTarget</var>.<dfn data-dfn-for="CropTarget">[[\Element]]</dfn> initialized to <var>element</var>.</p>
```

Comment:
> In any case, the following approach seems the most promising to me ...

I agree.

> ...and this PR is based doing that.
> 
> * Introduce a slot storing a weak reference to the element.

We should reflect this in the algorithm then. I think that's the confusion. How about:

```suggestion
              <p>Let <var>weakRef</var> be a weak reference to <var>element</var>.</p>
              <p>Create <var>cropTarget</var>.<dfn data-dfn-for="CropTarget">[[\Element]]</dfn> initialized to <var>weakRef</var>.</p>
```